helicopyter.py

```diff
@@ -127,29 +130,40 @@ def multisynth(all_or_conas_or_paths: Iterable[str]) -> None:
             cona = path_to_check.parent.parent.name
         else:
             cona = cona_or_path
-        relative_path = f'deploys/{cona}/terraform/main.tf.json'
+        relative_path = (
+            f'deploys/{cona}/terraform/main.tf{"" if hashicorp_configuration_language else ".json"}'
+        )
         print(f'Generating {relative_path}')
         module = import_module(f'deploys.{cona}.terraform.main')
         stack = HeliStack(cona)
         module.synth(stack)
-        dictionary = stack.to_terraform()
-        dictionary['//']['AUTOGENERATED'] = 'by helicopyter'
-        if stack.imports:
-            dictionary['import'] = [
-                {'to': resource, 'id': import_id} for resource, import_id in stack.imports.items()
-            ]
-        with (Path(__file__).parent / relative_path).open('w') as output:
-            dump(dictionary, output, indent=4, sort_keys=True)
-            output.write('\n')
+        if hashicorp_configuration_language:
+            unformatted = stack.to_hcl_terraform()['hcl'].encode()
+            formatted = check_output(['terraform', 'fmt', '-'], input=unformatted).decode()  # noqa: S603
+            (Path(__file__).parent / relative_path).write_text(
```

Comment:
I vote we move this `Path(__file__).parent` to the module level, and just call it `HELICOPYTER_DIR` or something

Reply:
Good call. It's used three times. https://wiki.c2.com/?ThreeStrikesAndYouRefactor